<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Knapsack</class>
 <widget class="QWidget" name="Knapsack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_bonds">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>460</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>点券</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/Bonds.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bonds">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>455</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>点券</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>441</y>
     <width>831</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/BlueLine.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="MouseEvent" name="label_shop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>商城</string>
   </property>
   <property name="accessibleName">
    <string>商城</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/Shop.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Img.qrc">:/Img/Btn_AddFriends.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>831</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文彩云</family>
     <pointsize>45</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>背包</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_renamecard">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_renamecardnumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>拥有:2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MouseEvent</class>
   <extends>QLabel</extends>
   <header>mouseevent.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Img.qrc"/>
 </resources>
 <connections/>
</ui>
